_vcsh_bash

```diff
@@ -128,11 +132,17 @@ _vcsh () {
 
 	# git command on repository
 	if [[ -n "${repos[$cmd]}" ]]; then
-		_vcsh_git_command $subcword
+		local REPO_DIR="${XDG_CONFIG_HOME:-$HOME/.config}/vcsh/repo.d/${cmd}.git"
+		GIT_DIR=${REPO_DIR} _vcsh_git_command $subcword
 	fi
 	return 0
 }
 
 complete -F _vcsh vcsh
 
 # vim: ft=sh:
+# Local Variables:
+# mode:shell-script
+# sh-shell:bash
+# indent-tabs-mode: t
```

Comment:
I'm okay with specifying the file type hints in a modeline, but lets leave the indentation hints for Editor Config. See #237 for where we just recently added that.

Reply:
Understood. Will remove that the tab setting.